src/EditorFeatures/Core/Implementation/Classification/SemanticClassificationViewTaggerProvider.cs

```diff
@@ -98,7 +98,7 @@ protected override Task ProduceTagsAsync(TaggerContext<IClassificationTag> conte
             if (_classificationService == null)
             {
                 var document = spanToTag.Document;
-                _classificationService = document?.Project.LanguageServices.GetService<IEditorClassificationService>();
+                _classificationService = document?.Project?.LanguageServices?.GetService<IEditorClassificationService>();
```

Comment:
If you have a document, then it must have a project, so Project can't be null.  Same with LanguageServices.  The .?s here should be unnecessary.